final_task/rss_reader/article.py

```diff
@@ -1,45 +1,40 @@
 from string_operations import *
 import logging
-
+import datetime
 
 class Article:
     """single article class"""
 
-    def __init__(self, parsed):
+    def __init__(self, parsed, source):
         """receive parsed article and extracts data from it"""
-        logging.info('      title')
         self.title = make_string_readable(parsed.title)
-
-        logging.info('      link')
         self.link = parsed.link
-
-        logging.info('      date')
+        self.feed_link = source
         self.published = extract_date(parsed)
 
-        logging.info('      summary')
         summary_ = extract_topic_info_from_summary(parsed.summary)
         self.summary = make_string_readable(summary_)
-
-        logging.info('      image links')
         self.media = parsed.media_content
 
-        logging.info('      image description')
         description_ = extract_image_info_from_summary(parsed.summary)
         self.media_description = make_string_readable(description_)
 
     def print_readable_article(self):
         """print article to stdout in human-readable format"""
-
-        # separation line
         print("_" * 79)
+        date = self.published
+        print('{}/{}/{}, {}:{}:{}\n'.format(
+            date.tm_year,
+            date.tm_mon,
+            date.tm_mday,
+            date.tm_hour,
+            date.tm_min,
+            date.tm_sec
+        ))
```

Comment:
It is better to use `formatted string` (`f"..."`) instead of `.format`

Reply:
Fixed